projects/js-packages/shared-extension-utils/changelog/fix-upsell-nudge-safari

```diff
@@ -0,0 +1,4 @@
+Significance: patch
```

Comment:
Since we're effectively removing something entirely from the package and since that package is published to npm, I think a major version bump may be more appropriate here, so we don't accidentally break things for folks who may use the package and who may not expect functionality to be removed with a patch release.

Reply:
We're just removing code, no? It's not a user facing thing that is removed? Let me know if you still think `major` is appropriate here.

Reply:
We're only removing code indeed, but that changelog is not aimed at site owners. It is aimed at consumers of the `@automattic/jetpack-shared-extension-utils` package, i.e. developers. They may be interested in knowing that a function they may have been using will be removed entirely from the package. 

I would personally consider this as a major change for consumers of the package, so they'll know to be mindful before they update the package version in their apps.

Reply:
Thanks for the update. I just pushed 241b062926c37f777d6050612fb63a9341b28c43 as a final tweak, so the major bump only happens in the package where we have the major change. In the other packages, we're not introducing any change that would break backward compatibility.

Reply:
Thank you!